addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/classes/Meta.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.neato.internal.classes;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Meta {
+
+    @SerializedName("modelName")
```

Comment:
Maybe better leave out the @SerializedName on the places where it matches and I think expose is only use for the other way around, constructing an object and passing it to an webservice, so it can be removed. Please apply to all classes.